csharp/src/Apache.Arrow/Arrays/Time32Array.cs

```diff
@@ -0,0 +1,138 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one or moreDate32Array
+// contributor license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Apache.Arrow.Types;
+using System;
+using System.IO;
+
+namespace Apache.Arrow
+{
+    /// <summary>
+    /// The <see cref="Time32Array"/> class holds an array of ints, where each value is
+    /// stored as the number of seconds/ milliseconds (depending on the Time32Type) since midnight.
+    /// </summary>
+    public class Time32Array : PrimitiveArray<int>
+    {
+        private static readonly int seconds_in_millisecond = 1_000;
```

Comment:
[nits]
How about directly using `1_000` instead of this field as in `Time64Array` ?
I think a meaning of `1_000` is clear in `GetSeconds()` and `GetMilliSeconds()`.

Reply:
made this change